oscrypto/_ffi.py

```diff
@@ -386,6 +390,35 @@ def callback(library, signature_type, func):
     engine = 'ctypes'
 
 
+def get_library(name, dylib_name, version):
+    """
+    Retrieve the C library path with special handling for macOS.
+
+    :param name:
+        The library to search the system for.
+    :param dylib_name:
+        The expected unversioned dylib name.
+    :param version:
+        dylib version override.
+        Preferred for macOS 15.15+
+        and used as a fallback for 10.16 where `find_library` doesn't work.
+    :return:
+        Path to the library.
+    """
+    library = find_library(name)
+    if sys.platform == 'darwin':
+        unversioned = '/usr/lib/{}'.format(dylib_name)
```

Comment:
From the tests it would appear this code doesn't work on Python 2.6. The test of the code base pretty much uses `%`, so I'd probably stick with that.

Same for the line below.

Reply:
Indeed, empty formatting specifiers without an index are only supported on Python 3.

Reply:
> Indeed, empty formatting specifiers without an index are only supported on Python 3.

That can't be true, can it? I was definitely using those on 2.7.x with IronPython, and maybe 2.6.x.

Reply:
Okay, so not 2.6, but [the docs](https://docs.python.org/2.7/library/string.html#formatstrings) _do_ say 2.7:

> _Changed in version 2.7:_ The positional argument specifiers can be omitted for `str.format()` and `unicode.format()`, so `'{} {}'` is equivalent to `'{0} {1}'`, `u'{} {}'` is equivalent to `u'{0} {1}'`.

Reply:
Ah yes, I made that mistake a while back already. Pretty much all the Python 2 code I ever wrote was for Python 2.6, so I mix some of these limitations.